### PR TITLE
*: update tidb

### PR DESCRIPTION
**Pull request by disksing**
### What problem does this PR solve? 
tidb dependency is too old. Need to update.

### What is changed and how it works?
- update modules.
- migrate ddl updates

### Check List 

Tests 
 - Unit test

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
LGTM

File: pkg/dbutil/table.go [hunk 1]
```diff
@@ -52,202 +50,30 @@ func GetTableInfoBySQL(createTableSQL string, sqlMode string) (table *model.Tabl
...
-				// FIXME: use a default string now to make col.IsGenerated() return true, will use real generated expr string later
-				col.GeneratedExprString = "Generated"
-				// do nothing
-			case ast.ColumnOptionFulltext:
-				// do nothing
+		// put primary key in indices
+		if table.PKIsHandle {
+			pkIndex := &model.IndexInfo{
+				Name:    model.NewCIStr("PRIMARY"),
+				Primary: true,
```
Comment (zimulala) [inline pkg/dbutil/table.go:62]: Do we need to set the values of `State` and `Unique`?

Comment (WangXiangUSTC) [reply pkg/dbutil/table.go:62]: set in 17be9e7

**Review by zimulala — APPROVED** [review]
LGTM

**Pull request merged**